cmp/cmpopts/equate.go

```diff
@@ -123,3 +124,36 @@ func (a timeApproximator) compare(x, y time.Time) bool {
 	// Note: time.Time doesn't have AfterOrEqual method hence the negation.
 	return !x.Add(a.margin).Before(y)
 }
+
+// AnyError is an error that matches all other non-nil errors.
```

Comment:
AnyError matches itself as well, so perhaps "...that matches any non-nil error."?

Reply:
Done.